.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.16.3
+    steps:
```

Comment:
There's a way to set up a cache to speed up circle ci for future builds, I believe it's written something like so:

```
   steps:
      - restore_cache:
          keys:
            - v1-dependencies-{{ checksum "package.json" }}
            # fallback to using the latest cache if no exact match is found
            - v1-dependencies-
```